api/controllers/user.controller.ts

```diff
@@ -1,54 +1,74 @@
-import { Request, Response } from 'express';
+import { Request, Response, NextFunction } from 'express';
 import UserService from '../secvices/user.service';
 
 class UserController {
-    public async get(req: Request, res: Response): Promise<void> {
-        const { id } = req.params;
-        const userService = new UserService();
+    public async get(req: Request, res: Response, next: NextFunction): Promise<void> {
+        try {
+            const { id } = req.params;
+            const userService = new UserService();
 
-        const user = await userService.getById(+id);
+            const user = await userService.getById(+id).catch(next);
```

Comment:
Why do you need to catch error here, as it'll be caught because of `try...catch` block and `async-await` usage?

Reply:
Fixed